### PR TITLE
17 update readme

### PR DESCRIPTION
**Pull request by cstanger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,110 @@
...
+If you click on any of the icons the page will show the detailed view of the selectedservice.
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+![Cloud Landscape Detail View](./docs/screenshots/detailView.png)
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Using your own data
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+In order to you use your own data source (preferable a none-SQL Database), each document in your database must be structured as follows. In the provided example these are the cloud service providers (e.g. Amazon) and the actual cloud service category (e.g. “Storage”). Every entry must match the following JSON format:
```
Comment (elKei24) [inline README.md:56]:
```suggestion
In order to you use your own data source (preferable a non SQL Database), each document in your database must be structured as follows. In the provided example these are the cloud service providers (e.g. Amazon) and the actual cloud service category (e.g. “Storage”). Every entry must match the following JSON format:
```

File: README.md [hunk 2]
```diff
@@ -1,49 +1,110 @@
...
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+At the top of the page you will find the search input. The search runs using the logical “AND” so an item has to match all of the entered keywords. In order to zoom the contents of the page use the zoom buttons to the left of the search bar. You can either zoom everything (left toolbox) or only the text (right toolbox). Press the button between the two toolboxes to reset the zoom.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+![Cloud Landscape](./docs/screenshots/landscapeView.png)
+![Cloud Landscape Filter](./docs/screenshots/filter.png)
 
-### `npm run eject`
+If you click on any of the icons the page will show the detailed view of the selectedservice.
```
Comment (elKei24) [inline README.md:50]:
```suggestion
If you click on any of the icons the page will show the detailed view of the selected service.
```

File: README.md [hunk 3]
```diff
@@ -1,49 +1,110 @@
...
+- [Getting started](#Getting-started)
+- [Using your own data](#Using-your-own-data)
+- [Further dependencies](#Further-dependencies)
+- [Used libraries](#Used-libraries)
+- [License](#License)
+
+## What is Cloud Landscape
 
-## Available Scripts
+Cloud Landscape is a web-based visualization tool. It was initially designed to visualize the set of cloud services provided by Microsoft (Azure), Amazon (AWS) and Google (GCP). This project includes the respective database for demonstration purposes. Our Cloud Landscape tool also provides a basic search functionality for the items in your database as well as the possibility to show connections between items. In the context of cloud services these connections represent service dependencies for different scenarios.
```
Comment (elKei24) [inline README.md:21]: Are we going to implement the dependency connections in the near future? If not, I would suggest not mentioning them for now.

Comment (cstanger) [reply README.md]: I don't think so - and if we can easily update the readme - I will remove them

**Pull request merged**